CHANGELOG.md

```diff
@@ -6,6 +6,49 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 The crates in this repository do not adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) at this time.
 
+## [6.0.2]
+
+### Fixed
+
+- Fix potentional deadlock in mobilecoind ([#3986])
+
+[#3986]: https://github.com/mobilecoinfoundation/mobilecoin/pull/3986
+
+## [6.0.1]
```

Comment:
I realized I forgot to break out the 6.0.1 changes to their own section
The PR that updated 6.0.1 for reference, https://github.com/mobilecoinfoundation/mobilecoin/pull/3985